lib/cli.js

```diff
@@ -88,6 +90,16 @@ exports.run = function () {
 	}
 
 	if (
+		hasFlag('--no-color') ||
+		conf.noColor
+	) {
+		chalk.enabled = false;
+		Object.keys(colors).forEach(function (key) {
+			colors[key].enabled = false;
+		});
```

Comment:
Actually. None of this should be needed as `chalk` already supports the `--no-color` flag.